src/app/page.js

```diff
@@ -1,95 +1,96 @@
-import Image from "next/image";
-import styles from "./page.module.css";
+// import Image from "next/image";
```

Comment:
I would suggest to use scss and default imports, but you use css modules
In that case you will have more controlle with your styling !!